cmd/opampsupervisor/supervisor/config/config.go

```diff
@@ -123,6 +123,7 @@ type Agent struct {
 	Description             AgentDescription `mapstructure:"description"`
 	BootstrapTimeout        time.Duration    `mapstructure:"bootstrap_timeout"`
 	HealthCheckPort         int              `mapstructure:"health_check_port"`
+	Containerized           bool             `mapstructure:"containerized"`
```

Comment:
Will this option change other functionality related to the Collector process? If it is only for re-emitting the Collector's logs, I think we should be more specific with the name. I see why the name makes sense having read through everything, but my first guess on seeing this is that it would affect more than just the log output.

Reply:
No it doesn't change any other functionality related to the collector so that's a great point. I could do `passthrough_logging` to be more specific about what it does. Does that name work or is there another you have in mind?

Reply:
That name sounds good to me.

Reply:
A few less characters to type: `passthrough_logs`?